cartridge/shop/templates/admin/shop/product/change_list.html

```diff
@@ -0,0 +1,26 @@
+{% extends "admin/change_list.html" %}
+{% load shop_tags pages_tags i18n staticfiles %}
+
+{% block extrahead %}
+{{ block.super }}
+
+<script src="{% static "mezzanine/js/jquery-ui-1.8.14.custom.min.js" %}"></script>
+<script src="{% static "mezzanine/js/admin/page_tree.js" %}"></script>
+{% endblock %}
```

Comment:
I spoke too soon. page_tree.js is what makes makes navigation to a new product form happen when you make the product type selection.